src/explorer.api/Explorers/MinMaxExplorer.cs

```diff
@@ -37,21 +37,30 @@ private async Task RefinedEstimate(bool isMin, CancellationToken cancellationTok
         {
             var estimator = isMin ? (Estimator)GetMinEstimate : (Estimator)GetMaxEstimate;
 
-            decimal? estimate;
-            decimal? result = null;
-
-            estimate = await estimator(result, cancellationToken);
+            // initial unconstrained min or max
+            var result = await estimator(null, cancellationToken);
 
+            // limit the number of iterations
             for (var i = 0; i < MaxIterations; i++)
             {
-                result = estimate;
-                estimate = await estimator(result, cancellationToken);
+                // If it'a a minimum and we have a zero result, it can't be improved upon anyway.
+                if (isMin && result == decimal.Zero)
```

Comment:
This is not clear to me: can't we have a negative minimum?

Reply:
Yes, I think it's possible, and I think the min refinement explorer would fail if the column has negative values. However if I remove this condition, the explorer tries to send a query with `{column} between (0 and 0)` and this causes an error. I'm not sure how best to deal with negative values... 

I logged an issue for this: #91.